src/modules/model.js

```diff
@@ -147,21 +155,55 @@ class Model {
     await this._callback('delete', id);
   }
 
-  async list({ indexName, indexMap, fields }) {
+  async list({
+    indexName,
+    indexMap,
+    fields,
+    limit = 20,
+    cursor = null
+  }) {
     // eslint-disable-next-line no-underscore-dangle
     this._before();
+    const splitFields = objectFields.split(fields);
+    const fieldsContainsId = splitFields.includes('id');
+    if (!fieldsContainsId) {
+      splitFields.push('id');
+    }
+
+    const {
+      lastEvaluatedKey,
+      scanIndexForward = true,
+      limit: queryLimit = limit,
```

Comment:
Awkward, in that we haven't done param renaming with a default before. I don't necessarily think it's bad, feel free to disagree though :)